docs/milestones.md

```diff
@@ -0,0 +1,31 @@
+# Milestones 
+
+---
+
+## [M0] Implementado del Modelo de Entidades en Código
+
+- **Producto a entregar:**  
+	Código que implementa un modelo de entidades clave de la Sierra de Segura. El código debe permitir definir las relaciones entre las entidades, como la relación entre un municipio y su población, asegurando que el modelo sea extensible y compatible con una futura base de datos.
```

Comment:
Por favor, hablad siempre explícitamente de las historias de usuario con las que estéis trabajando. Incluso aunque sea una, no cuesta ningún trabajo mencionarla (además, en ágil pueden añadirse HUs en cualquier momento)